src/content/docs/crowdin/project-management/sources/version-management.mdx

```diff
@@ -66,37 +142,75 @@ To set a priority for your branches, follow these steps:
+When this option is applied, only the original strings (usually from the master branch) need to be translated. All duplicates in other branches automatically receive translations from those originals.
 
-<Image src={versionsDuplicateStrings} alt="Duplicates"/>
+<Tabs syncKey="projectType">
+  <TabItem label="File-based project">
+    In file-based projects, you can configure how duplicates are handled in **Settings > Import > Source Strings**. If your source files contain clear identifiers (keys), it's recommended to use the *Strict* version of the **Show within a version branch** option. Otherwise, the *Regular* option will work well for most use cases.
+  <Image src={versionsDuplicateStrings} alt="Duplicates"/>
+  </TabItem>
+  <TabItem label="String-based project">
+    In string-based projects, the **Show within a version branch (regular detection)** behavior is applied automatically and cannot be changed. This ensures translations are reused between branches without requiring any additional configuration.
+  </TabItem>
+</Tabs>
 
 <ReadMore>
-  Read more about [Duplicates settings](/project-settings/import/).
+  Read more about [Duplicates settings](/project-settings/import/#duplicate-strings).
 </ReadMore>
 
 ## Suggested Workflow
 
-To make sure versions work smoothly for your Crowdin project, we recommend the following workflow:
+To ensure smooth collaboration between developers and translators, version branches in Crowdin can be used in different ways depending on how you manage source content and what project type you're using.
 
-1. Upload your project files to the Crowdin project using one of the [available methods](#branch-creation).
-2. Open your project and go to **Settings > Import > Source Strings**.
-3. Switch the **Duplicate Strings** option to **Show within a version branch**.
+### Managing Content Outside Crowdin
 
-The screenshot below visualizes how the workflow works in practice. All texts from *Master*, *Branch 1*, and *Branch 2* are transferred to the translation server immediately after they appear, even though the branches are not merged to the *Master* branch yet.
+In both file-based and string-based projects, you can treat your external system (e.g., file storage or version control) as the source of truth. In this setup:
+
+* Source files are managed and updated outside of Crowdin.
+* You upload project content using the web interface (UI), API, CLI, or integrations.
+* Selected branches are created in Crowdin accordingly.
+* Translations are delivered back to your system once completed.
+
+<Steps>
+  1. Upload your project files to the Crowdin project using one of the [available methods](#creating-branches).
+  2. In file-based projects, open your project and go to **Settings > Import > Source Strings**.
+  3. Set the **Duplicate Strings** option to **Show within a version branch**.
+</Steps>
```

Comment:
There is something wrong with this part. The steps component should have some description before it. It's hard to figure out how it relates to the list.